src/ui/screens/ScreenOnboarding.qml

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Controls 2.14
+
+import Mozilla.Shared 1.0
+import Mozilla.VPN 1.0
+import components 0.1
+
+MZScreenBase {
+
+    objectName: "screenOnboarding"
+    Component.onCompleted: () => {
+        MZNavigator.addStackView(VPN.ScreenOnboarding, getStack())
+        getStack().push("qrc:/ui/screens/onboarding/ViewOnboarding.qml")
+    }
+    Connections {
+        target: MZTutorial
+        function onPlayingChanged() {
+           if (MZTutorial.playing) {
+               getStack().pop(null, StackView.Immediate)
+           }
+        }
+    }
+    Connections {
+        target: window
+
+        // TODO - This only works when we are already in ScreenHome
```

Comment:
Is there a followup issue for this TODO? Or will it be implemented as part of this effort?

Reply:
Ahhh, this was a bad copy and paste (from `ScreenHome`) this code is unnecessary in `ScreenOnboarding`